ardupilot/equipment/power/20008.SmartBatteryContinuous.uavcan

```diff
@@ -0,0 +1,22 @@
+#
+# Smart battery data to be sent continuously (10Hz or greater)
+#
+
+int16 temperature             # [cdegC]
+int16 current                 # [cA]
```

Comment:
float16 only gives slightly more range, I am not aware of any systems that draw more than 655A but I am sure this could be an issue as aircraft sizes grow. Is it better to plan for the future or to try and optimize bytes? I am not opposed to an extra few bytes

Reply:
Yeah me neither at this point. But there are ESCs that can individually draw 400A so it is not inconceivable to me to have a battery system as a whole report more than 600A.

Reply:
yeah I agree, I bumped it up to a float32